.github/workflows/build-and-test.yml

```diff
@@ -25,33 +25,584 @@ jobs:
+        with:
+          fetch-depth: 0
+          submodules: 'recursive'
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+      - run: |
+          sudo apt-get install -y --no-install-recommends pixz
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+      - uses: ./.github/actions/setup-tensorflow
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+      - uses: ./.github/actions/build-tensorflow
+        with:
+          flavor: "--linux-cpu"
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+      - uses: ./.github/actions/package-tensorflow
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ needs.tensorflow_opt-Linux.outputs.cache_key }}
+          path: ${{ github.workspace }}/artifacts/home.tar.xz
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+  build-lib_Linux:
+    name: "Lin|Build libdeepspeech+client"
+    runs-on: ubuntu-20.04
+    needs: [ build-tensorflow-Linux, tensorflow_opt-Linux ]
+    strategy:
+      matrix:
+        build-flavor: ["tf", "tflite"]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/download-artifact@v2
+        with:
+          name: ${{ needs.tensorflow_opt-Linux.outputs.cache_key }}
+          path: ${{ github.workspace }}/
+        if: needs.tensorflow_opt-Linux.outputs.status == 'missing'
+      - uses: ./.github/actions/check_artifact_exists
+        with:
+          name: ${{ needs.tensorflow_opt-Linux.outputs.cache_key }}
+          path: ${{ github.workspace }}/
+          download: true
+        if: needs.tensorflow_opt-Linux.outputs.status == 'found'
+      - run: |
+          tar --skip-old-files -xf ${{ github.workspace }}/home.tar.xz
+          rm ${{ github.workspace }}/home.tar.xz
+      - run: |
+          sudo apt-get install -y --no-install-recommends make build-essential gfortran git libblas-dev liblapack-dev libsox-dev libmagic-dev libgsm1-dev libltdl-dev libpng-dev python python-dev zlib1g-dev
+      - run: |
+          git status
+      - uses: ./.github/actions/host-build
+        with:
+          flavor: ${{ matrix.build-flavor }}
+      - uses: ./.github/actions/package
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "native_client.${{ matrix.build-flavor }}.Linux.tar.xz"
+          path: ${{ github.workspace }}/artifacts/native_client.tar.xz
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "libdeepspeech.${{ matrix.build-flavor }}.zip"
+          path: ${{ github.workspace }}/artifacts/libdeepspeech.zip
+  build-python-Linux:
+    name: "Lin|Build Python bindings"
+    runs-on: ubuntu-20.04
+    needs: [ build-lib_Linux, swig_Linux ]
+    strategy:
+      matrix:
+        build-flavor: ["tf", "tflite"]
+        python-version: [3.6.13, 3.7.10, 3.8.9, 3.9.4]
```

Comment:
nit: it looks like `setup-python` hits more frequently api rate limit when requesting finer grained version than just X.Y